pkg/controller/daemon/daemoncontroller.go

```diff
@@ -579,23 +735,37 @@ func (dsc *DaemonSetsController) manage(ds *extensions.DaemonSet) error {
 			}
 		}
 	}
-	errors := dsc.syncNodes(ds, podsToDelete, nodesNeedingDaemonPods)
+
```

Comment:
The previous blocks in this function have good comments.  This new code would benefit from them too.